README.md

```diff
@@ -138,6 +138,13 @@ public class StringExample {
 
 You can find more examples in the [examples directory][examples].
 
+## Public API
+
+The public API that we will maintain within the constraints of our [versioning policy](./VERSIONING.rst)
+consists of all public classes in the `com.amazonaws.encryptionsdk` package unless otherwise and specifically documented.
+
```

Comment:
The public API that we maintain includes all public classes in the in the `com.amazonaws.encryptionsdk` package unless otherwise documented. Our [versioning policy](./VERSIONING.rst) applies to this public API.

Reply:
Edit: "Our [versioning policy](./VERSIONING.rst) applies to all public classes in the `com.amazonaws.encryptionsdk`
package unless otherwise documented."

Reply:
Also, not to nitpick: it also applies to public constants, methods, etc.

Reply:
... and protected methods, constants, etc.

Reply:
Second try: Our versioning policy applies to all public and protected classes/methods/fields in the com.amazonaws.encryptionsdk
package unless otherwise documented.